packages/compass-connections/src/components/legacy-connections.spec.tsx

```diff
@@ -42,7 +42,7 @@ async function loadSavedConnectionAndConnect(connectionInfo: ConnectionInfo) {
 
   // Wait for the connection to load in the form.
   await waitFor(() =>
-    expect(screen.queryByRole('textbox')?.textContent).to.equal(
+    expect(screen.queryByTestId('connectionString')?.textContent).to.equal(
```

Comment:
This kind of testing breaks the moment there are multiple textboxes on screen. But also in this case I guess there shouldn't be because it should be rendering the legacy connections form..